src/Layout3View.js

```diff
@@ -63,7 +65,7 @@ const styles = StyleSheet.create({
   </View>
 </View> */}
 
-const MyDocument = () => (
+const Layout3View = () => (
```

Comment:
i think i got it. see at function arguments? we did't pass `data` as props.

i assume if we'll change it to `Layout3View = ({ data }) => (` it maybe will work

Reply:
const Layout3View = ({ data }) => ( 
doesn't work

Reply:
ok, commit your update and i'll take a look

Reply:
don't worry i'm fetching your forked repository